src/Conversions.ts

```diff
@@ -0,0 +1,19 @@
+import { toBN, fromWei, toWei } from 'web3-utils';
+/**
+ * TODO: Hard-coded values: for testing purposes only!
```

Comment:
also, do we have a JIRA for this? I can only find PP-106 but that one is completed

Reply:
No, we don't. Although we documented it among the other steps required, we haven't created a JIRA task yet.

Reply:
thank you 🙏🏼